### DISCUSSION TITLE
Consistency in definition for retrieving incoming and outgoing edges/neighbours

### DISCUSSION THREAD

**Discussion by general-rishkin** [fslaborg/Graphoscope#40] [Discussions]
For a `DiGraph` **G**, to retrieve the incoming neighbours of a node (say, node 1), you write:

`1 |> DiGraph.getInEdges G `

whereas to retrieve the outgoing neighbours, you write:

`G |> DiGraph.getOutEdges 1`

It would be more consistent to select either:

```
1 |> DiGraph.getInEdges G
1 |> DiGraph.getOutEdges G
```

OR 

```
G |> DiGraph.getInEdges 1
G |> DiGraph.getOutEdges 1
```

**Comment by LibraChris** [comment]
Converted this into issue #41  and solved this by aligning the Parameter Order for DiGraph.getInEdges with DiGraph.getOutEdges (The second variant, assigning the graph as the pipeable parameter.
